libcontainer/integration/seccomp_test.go

```diff
@@ -60,7 +60,7 @@ func TestSeccompDenyGetcwdWithErrno(t *testing.T) {
 	}
 	ps, err := pwd.Wait()
 	if err == nil {
-		t.Fatal("Expecting error (negative return code); instead exited cleanly!")
+		t.Fatalf("Expecting error (negative return code); instead exited cleanly!")
```

Comment:
This hunk is useless; `t.Fatal` is more suitable here.

Reply:
Yep thanks for spotting this, I changed it while debugging :) !